mlir/test/Dialect/Vector/vector-gather-lowering.mlir

```diff
@@ -74,6 +74,32 @@ func.func @gather_memref_1d_i32_index(%base: memref<?xf32>, %v: vector<2xi32>, %
   return %0 : vector<2x3xf32>
  }
 
```

Comment:
I think this is missing a negative test (with a type like `[4]x8`) where there's no unrolling possible.